src/static/provider_apikeyid.merged.json

```diff
@@ -0,0 +1,7 @@
+[
+    {
+        "id": "foobar",
```

Comment:
I think we can either remove this file entirely or add a comment inside to give some context (it isn't actually used like this but is instead populated during deployment)

Reply:
Updated the file to provide more description of how it is used - https://github.com/Notarise-gov-sg/api-notarise-healthcerts/pull/610/commits/9b02189d49cfbc6b41229456032b90e8add10109

Would it be better to keep the file so that people can refer to the structure in the future?